i18n-tasks.gemspec

```diff
@@ -38,7 +38,7 @@ TEXT
   s.add_dependency 'activesupport', '>= 4.0.2'
   s.add_dependency 'ast', '>= 2.1.0'
   s.add_dependency 'easy_translate', '>= 0.5.0'
-  s.add_dependency 'erubis'
+  s.add_dependency 'erubi'
```

Comment:
Should we limit the version of `erubi`?
Like `>= 1.0.0` or `~> 1.0`, like what these project do
- [rails](https://github.com/rails/rails/pull/27757/files#diff-eeca8f49241a2220a7446177d9f1c919)
- [better_error](https://github.com/charliesome/better_errors/pull/390/files#diff-bb262a96ce72496f61bd45636edbc2a9)